Essentials/src/messages.properties

```diff
@@ -102,6 +102,7 @@ destinationNotSet=Destination not set\!
 disabled=disabled
 disabledToSpawnMob=\u00a74Spawning this mob was disabled in the config file.
 disableUnlimited=\u00a76Disabled unlimited placing of\u00a7c {0} \u00a76for {1}.
+disposal=\u00a7cDisposal
```

Comment:
Should this have a colour by default? It didn't have a colour before.

Reply:
Ok, but we need GUI formatting like colors. On the sign, there is no need for the translated string, so we use it without it.

Reply:
I was referring to the inventory title created here: https://github.com/EssentialsX/Essentials/blob/849efa9756495a34260d57ff1d81df1283e5f5e7/Essentials/src/com/earth2me/essentials/signs/SignDisposal.java#L14

Reply:
I was referring to the inventory title created here: https://github.com/EssentialsX/Essentials/blob/849efa9756495a34260d57ff1d81df1283e5f5e7/Essentials/src/com/earth2me/essentials/signs/SignDisposal.java#L14

The title of the inventory was the default before - we should keep the normal grey colour (no colour code) as the default.